examples/minimal/packages/contracts/mud.config.ts

```diff
@@ -1,7 +1,8 @@
-import { mudConfig } from "@latticexyz/world/register";
+import { mudConfig } from "@latticexyz/world/snapSync";
```

Comment:
this approach looks like it wouldn't allow for composability of plugins

@dk1a is this how we're expected to create/use plugins?

Reply:
this should work well, and it's the approach I use for `world` the plugin. It just means this plugin can't be used without `store` and `world`, and implicitly registers those so you can skip em (or you can still import em).
Here I guess it just looks a bit weird/misleading

Reply:
maybe keep `import { mudConfig } from "@latticexyz/world/register";` and have snapSync be 2nd?